execute/table/chunk.go

```diff
@@ -108,9 +110,9 @@ func (v Chunk) Ints(j int) *array.Int {
 	return v.Values(j).(*array.Int)
 }
 
-// Uints is a convenience function for retrieving an array
+// UInts is a convenience function for retrieving an array
 // as a uint array.
-func (v Chunk) Uints(j int) *array.Uint {
+func (v Chunk) UInts(j int) *array.Uint {
 	return v.Values(j).(*array.Uint)
 }
```

Comment:
I have a vague memory of @scbrickley proposing this exact change a couple months ago and it was rolled back for some reason. Am I remembering it wrong? If not, maybe Sean has context.

Reply:
I think I wanted to change this (or some similar method) to match the [flux column type](https://github.com/influxdata/flux/blob/a42e5be988f19519b824807992c6846792dbbc8a/result.go#L87), but @jsternberg pointed out that the method was meant to match the [type from the `array` package](https://github.com/influxdata/flux/blob/a42e5be988f19519b824807992c6846792dbbc8a/array/builder.gen.go#L16).

I don't have a strong preference for either. Just weighing in with the context.

Reply:
`table.Chunk` is very intentionally not supposed to implement `flux.ColReader`. Part of that is because I didn't want to propagate the interface more as, in an ideal world, `flux.ColReader` would be gone. Part of it is because I wanted to normalize the method names to match the array package.

Similar to the `Times()` method below. `flux.ColReader` had code to panic if you accessed an array the wrong way which was awkward for times since they were just integers. This was intentionally changed.

Reply:
The code you gave me caused circular dependencies so this seemed like a simpler change :shrug:

Reply:
I think that might have been because the sample code I gave referenced the `table` package, but this is the `table` package. Which means that it referenced the internal one instead of itself and the internal one contains type aliases to this package. If you just remove the references to the `table` package, it seems to work. I'll put up a PR.

Reply:
That might be it, the error for circular dependencies are so awful I didn't realize...

Reply:
They are. Alternate PR is up here: https://github.com/influxdata/flux/pull/5090